site/content/entry/yuzu-progress-report-jun-2020/index.md

```diff
@@ -0,0 +1,174 @@
+
+## Dreams and Reality
+
+Project Maribel Hearn is out, and it is in Mainline too! Native 32 bit Switch games start to work, with some of them even being fully playable! This means that games like `New Super Mario Bros. U Deluxe` and `Captain Toad: Treasure Tracker` work almost perfectly and with great performance, while games like `Mario Kart 8 Deluxe` still need more work. 
+
+This has been a very requested feature for a very long time, but implementing it required a major rewrite of the kernel, and official support for ARMv7 was needed on [Dynarmic](https://github.com/MerryMage/dynarmic), our ARM JIT recompiler. This took a tremendous amount of work from several of our developers, including, but not limited to, [Merry](https://github.com/MerryMage), [Lioncache](https://github.com/lioncash), [Blinkhawk](https://github.com/FernandoS27), [bunnei](https://github.com/bunnei), [Rodrigo](https://github.com/ReinUsesLisp), [Tobi](https://github.com/FearlessTobi), [ogniK](https://github.com/ogniK5377), [Morph](https://github.com/Morph1984), and many others. Special mention goes to Merry and Lioncache: they are the soul of this project, and without them we couldn't have even started it. Several of their pull requests are "too technical" to be included in the progress reports, but it should be known that Merry and Lioncache are the main code contributors at the heart of yuzu.
+
+You may not noticed that four months ago, [bunnei](https://github.com/bunnei) and [Tobi](https://github.com/FearlessTobi) laid down the [initial framework](https://github.com/yuzu-emu/yuzu/pull/3478) towards 32 bit support on yuzu by refactoring the ARM interface to support 32 bit and 64 bit JIT backends, implementing a 32 bit ARM interface based on Citra's ARMv6 dynarmic backend by adding CP15 coprocessor support, and implementing several 32 bit SVCs such as `GetThreadPriority32`, `WaitSynchronization32`, and many more.
+
+While rewriting the kernel for Prometheus, [Blinkhawk](https://github.com/FernandoS27) and [bunnei](https://github.com/bunnei) saw an excellent opportunity to include support for 32 bit, ARMv7 CPU instructions and services. With this spark of motivation, they worked with [Merry](https://github.com/MerryMage) and [Lioncache](https://github.com/lioncash) to make the dreams of 32 bit a reality. By ramping up the implementation of various ARMv7 instructions such as `VPADD`, `VCVT` and `VSHL` on Dynarmic, and 32 bit SVCs such as `CreateTransferMemory32`, `MapSharedMemory32` and `CreateThread32` on yuzu. There are many other ARMv7 instructions and SVCs that we cannot cover here, but the culmination of this work allowed us to get the very first 32 bit games up and running! 
```

Comment:
Change `the dreams of 32 bit a reality.` to `the dream of 32-bit support a reality, `

make `By ramping` lowercase

Add comma after `VCVT`

Change `on Dynarmic` to `in Dynarmic`

Add comma after `MapSharedMemory32`

Change `on yuzu` to `in yuzu`